### DISCUSSION TITLE
My verification is not working

### DISCUSSION THREAD

**Discussion by mdmahinpro** [community/community#65470] [GitHub Education]
### Hi everyone,

I was a GitHub pro user, My university doesn't give edu mail at the honors level. I applied with my Gmail and got an email. But later when I tried to verify my academic status as my pro subscription was expired it did not approve my school ID card. I tried other documents also. But the outputs are the same, It's showing : 
The image you selected does not appear to contain your school name. Ensure your image (photo or upload) is legible. We do not accept school logos. If you are taking a photo of a screen try to have it as large a resolution as possible. You may include multiple documents in your image. If your original document is not in English then you might take a photo of the original next to an English translation.
Please help me validate my identity. I am still a student.

**Comment by arafajs** [comment]
1. Make sure the image is clear and in focus.
2. Make sure the image is large enough so that the school name is easy to read.
3. Make sure the image includes your school name, not just the logo.
4. If your school name is in a different language, include an English translation in the same image.
5. Try submitting a different document, such as a transcript or enrollment verification letter.
6. Make sure the document is dated within the past year.
7. Make sure the document is from an accredited institution.

If you have tried all of these things and you are still not able to get your verification approved, you can contact GitHub Support for assistance.
